rpc/core/events.go

```diff
@@ -105,8 +106,8 @@ func Subscribe(ctx *rpctypes.Context, query string) (*ctypes.ResultSubscribe, er
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to parse query")
 	}
-
-	subCtx, cancel := context.WithTimeout(context.Background(), subscribeTimeout)
+	// must be less than the server's write timeout (see rpcserver.DefaultConfig)
+	subCtx, cancel := context.WithTimeout(ctx.Context(), 5*time.Second)
```

Comment:
What do you think about an exported (but documented) const `const SubscriptionTimeoutSec := 5` which you use here and in `rpc/core/mempool.go`. That way this will show up in the godoc and as a global documented constant?

Reply:
I don't quite understand the benefit of doing this

Reply:
More exposure on what callers can expect / how this behaves? Also if this every changes you only need to modify on place (the constant).